tlslite/keyexchange.py

```diff
@@ -294,10 +298,31 @@ def processServerKeyExchange(self, srvPublicKey, serverKeyExchange):
 
     def makeClientKeyExchange(self):
         """Create client key share for the key exchange"""
-        cke = super(DHE_RSAKeyExchange, self).makeClientKeyExchange()
+        cke = super(ADHKeyExchange, self).makeClientKeyExchange()
         cke.createDH(self.dh_Yc)
         return cke
 
+# the DHE_RSA part comes from IETF ciphersuite names, we want to keep it
+#pylint: disable = invalid-name
+class DHE_RSAKeyExchange(ADHKeyExchange):
+    """
+    Handling of ephemeral Diffe-Hellman Key exchange
+
+    NOT stable API, do NOT use
+    """
+    
+    def __init__(self, cipherSuite, clientHello, serverHello, privateKey):
+        super(DHE_RSAKeyExchange, self).__init__(cipherSuite, clientHello,
+                                                 serverHello)
+        self.privateKey = privateKey
+#pylint: enable = invalid-name
```

Comment:
`#pylint: enable = invalid-name` needs to be just after call to super class, otherwise pylint ignores it (yes, I know that it's picky)